src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -164,7 +164,7 @@ TEST_F(CommandRequestImplTest, OnTimeOut_StateCompleted_UNSUCCESS) {
   EXPECT_EQ(RequestState::kCompleted, command->current_state());
 }
 
-TEST_F(CommandRequestImplTest, OnTimeOut_StateAwaitingHMIResponse_SUCCESS) {
+TEST_F(CommandRequestImplTest, DISABLED_OnTimeOut_StateAwaitingHMIResponse_SUCCESS) {
```

Comment:
@pold500 You have disabled 6 unit tests, please add a description why each of the tests was disabled by either adding a TODO or creating a common description in commit message.